libraries/eosiolib/contracts/eosio/multi_index.hpp

```diff
@@ -841,6 +855,56 @@ class multi_index
          return *ptr;
       } /// load_object_by_primary_iterator
 
+      template<typename Lambda>
+      void modify_object(T& obj, name payer, Lambda&& updater) {
+         using namespace _multi_index_detail;
+
+         auto& objitem = static_cast<item&>(obj);         
+         eosio::check( objitem.__idx == this, "object passed to modify is not in multi_index" );
+
+         eosio::check( _code == current_receiver(), "cannot modify objects in table of another contract" ); // Quick fix for mutating db using multi_index that shouldn't allow mutation. Real fix can come in RC2.
+
+         auto secondary_keys = make_extractor_tuple::get_extractor_tuple(indices_type{}, obj);
+
+         uint64_t pk = _multi_index_detail::to_raw_key(obj.primary_key());
```

Comment:
IMHO if you declare "using namespace _multi_index_detail;" it doesn't make sense to declare it here again.

Reply:
fixed